test/svg-sprite.test.js

```diff
@@ -515,8 +515,7 @@ describe('svg-sprite', () => {
                             should(error).not.ok;
                             looksSame(
                                 previewImage,
-                                path.join(__dirname, `expected/png/view.html${testConfig.namespace}.png`)
-                                , (error, result) => {
+                                path.join(__dirname, `expected/png/view.html${testConfig.namespace}.png`), (error, result) => {
```

Comment:
That's the work for prettier. I've tried to install it but it has so many errors even in this file that I  immediately understood why don't you use it here :D
But it strongly recommended anyway.

Reply:
I don't like prettier and we already enforce lint + style with xo :)

BTW xo has a prettier option, but I'd rather not enable it, at least not until all PRs are merged.

Reply:
I don't like it too. But nevertheless it's a decent tool that automatically does such changes. So I strongly recommend to turn it on.
It's OK to wait until all PR are merged and I have a strategy to use prettier less painfully.
1. ignore all files using `.prettiergnore`
2. Including javascript files step-by-step `.prettiergnore`

Reply:
The effect is not the same. Prettier is more in style formatting. Eslint is more in code quality. This two packages together covering up most cases.
Xo is just `JavaScript/TypeScript linter (ESLint wrapper) with great defaults` as docs says. It has `prettier` integration to ease configuration. I think we really should enforce it to ease core reviewing in future :)

Reply:
TBH I don't care about prettier *at all*. I use xo and its style and I'm happy with it.

But like I said, we can revisit after all PRs have been addressed.